BDS/hydro_bds_edge_state_2D.cpp

```diff
@@ -370,7 +382,7 @@ BDS::ComputeConc (Box const& bx,
     const auto dlo = amrex::lbound(domain);
     const auto dhi = amrex::ubound(domain);
 
-    auto bc = pbc[icomp];
```

Comment:
A few lines down (L392/404) `bc` is used on the GPU

Reply:
Good catch. I made it explicit to access pbc there. I think my previous version worked because it was capturing the simple `bc` struct from device to host. The updated way probably avoids a copy.